components/atom/progressBar/src/ProgressBarCircle/Circle/index.js

```diff
@@ -88,6 +90,10 @@ Circle.propTypes = {
   strokeWidth: PropTypes.node.isRequired,
   /** boolean to activate/desactivate animations */
   withAnimation: PropTypes.bool,
+  /** When progress stroke is bigger than main one, it would be double in width  */
+  outerStrokeWidth: PropTypes.bool,
+  /** The shape of the end of line, it can be "round" or "square" */
+  strokeLineCaps: PropTypes.string.isRequired,
```

Comment:
`strokeLinecap` inside the component but `strokeLineCaps` as the name of the prop? any reason for that?